src/pages/inscription/getUserId.query.integration.ts

```diff
@@ -0,0 +1,16 @@
+import { getUserId } from './getUserId.query';
+import { publish } from '../../dependencies/eventStore';
+import { FauxUtilisateurInscrit } from '../../domain/FauxUtilisateurInscrit';
+import { resetDatabase } from '../../dependencies/__test__/resetDatabase';
+
+describe('getUserId', () => {
```

Comment:
Deux aspects manquent dans le nommage:
- Quand on retourne un booléen, préfixer par `is` 
- Ici, c'est spécifique à la fake authentification, ça devrait se voir. 

Je l'aurai appelée `isFauxUtilisateurIdAvailable`.

Par ailleurs, cette query ne sert pas pour l'affichage de la page inscription. J'aurais mis cette query dans le dossier `fakeAuth` comme `getFakeUsers`.